docs/stack.md

```diff
@@ -0,0 +1,15 @@
+---
+sidebar_position: 3
+---
+
+# Revolt Stack
+
+The diagram below shows how each component interacts with each other.
+
+<img src="/img/diagrams/Revolt Architecture.png" />
```

Comment:
this is missing quart.

Reply:
going to ignore this for now until quark goes live